### PR TITLE
Adding modules for buscoannotation/getbuscogene. Closes #33.

### LINKED ISSUE #33: new module: buscoannotation/getbuscogene
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by weaglesBio**
## PR checklist

Closes #33 

- [] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by priyanka-surana** [comment]
Same as the other PR. Please fix the failing tests.

File: modules/buscoannotation/getbuscogene/main.nf [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    tag "$meta.id"
+    label 'process_medium'
+
+    def version = '0.001-c5'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the buscogeneannotation process. Please use docker or singularity containers."
+    }
+    //container "quay.io/sanger-tol/buscogeneannotation:${version}"
+    container "gitlab-registry.internal.sanger.ac.uk/tol-it/software/docker-images-test/buscogeneannotation:0.001-c5"
```
Comment (priyanka-surana) [inline modules/buscoannotation/getbuscogene/main.nf:11]: Use version since it is defined.

File: modules/buscoannotation/getbuscogene/meta.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - "buscoannotation":
+      description: a tool kit to extract ancestral gene based on the busco result
+      licence: ["MIT"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - tsv:
```
Comment (priyanka-surana) [inline modules/buscoannotation/getbuscogene/meta.yml:17]: Name needs to match the input above.

File: modules/buscoannotation/getbuscogene/meta.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      type: file
+      description: TSV file
```
Comment (priyanka-surana) [inline modules/buscoannotation/getbuscogene/meta.yml:19]: Detailed description.

**Comment by weaglesBio** [comment]
nf-core modules linting had no errors when run locally.

**Comment by priyanka-surana** [comment]
EditorConfig errors need fixing.

**Review by priyanka-surana — APPROVED** [review]
LGTM 👍

**Pull request merged**